package.json

```diff
@@ -31,5 +31,10 @@
     "babel-cli": "~6.14.0",
     "babel-plugin-add-module-exports": "~0.2.1",
     "babel-preset-es2015": "~6.14.0"
+  },
+  "jspm": {
+    "main": "src/index.js",
```

Comment:
This should not be the way to go, the library lives in `lib/index.js`.
And should probably install superlogin-client through npm not git, that's where versioning is controlled.
This is why `lib/index.js` is the only solution, `src/index.js` doesn't exist there.